app/views/admin/resources/index.html.haml

```diff
@@ -0,0 +1,38 @@
+.row
+  .col-md-12
+    .page-header
+      %h1 Resources
+      %p.text-muted
+        Manage resources in conference
+- if @conference.resources.any?
+  .row
+    .col-md-12
+      %table.table.table-hover.datatable
+        %thead
+          %th Name
+          %th Left( Left/Total )
+          %th Used
+          %th Actions
+        %tbody
+          - @conference.resources.each do |resource|
+            %tr
+              =semantic_form_for(resource, url: admin_conference_resource_path(@conference.short_title, resource.id), html: { method: :put }) do |f|
+                %td
+                  = link_to(admin_conference_resource_path(@conference.short_title, resource.id)) do
+                    = resource.name
+                %td
+                  = resource_quantity_left(resource)
+                %td.col-sm-1.col-md-1
+                  = f.input :used, label: false
```

Comment:
I used `input` keeping in mind that we have used `input` throughout the app even if the input required is a number. The input field does not accept alphabets or characters as input here because of [`formtastic-bootstrap`](https://github.com/mjbellantoni/formtastic-bootstrap) I guess. Changing it to `number_field` would remove the styling as well.